src/Compilers/CSharp/Portable/Syntax/DestructorDeclarationSyntax.cs

```diff
@@ -56,7 +56,7 @@ public partial class SyntaxFactory
             SyntaxToken tildeToken,
             SyntaxToken identifier,
             ParameterListSyntax parameterList,
-            BlockSyntax body,
+            BlockSyntax? body,
```

Comment:
Same comment

Reply:
The node allows body to be null and the IDE calls this method with null body here: https://sourceroslyn.io/#Microsoft.CodeAnalysis.CSharp.Workspaces/CodeGeneration/ConstructorGenerator.cs,64
I am guessing this is used e.g. for metadata as source where declarations have no bodies.

Reply:
Ah, the metadata-as-source scenario makes sense. Thanks.